pandas/io/pytables.py

```diff
@@ -4875,16 +4853,3 @@ def select_coords(self):
             return self.coordinates
 
         return np.arange(start, stop)
-
-# utilities ###
-
-
-def timeit(key, df, fn=None, remove=True, **kwargs):
```

Comment:
Just to confirm this is dead code and not necessarily related to Panel right?

Reply:
Yes.  Good idea double-checking.